services/medco/libmedco/survey_store_test.go

```diff
@@ -0,0 +1,145 @@
+package libmedco_test
+
+import (
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	. "github.com/dedis/cothority/services/medco/libmedco"
+	"github.com/dedis/crypto/random"
+	"reflect"
+	"testing"
+)
+
+// TestStoring tests survey store and its methods.
+func TestStoring(t *testing.T) {
```

Comment:
General remark (no need to do anything): it is better to have smaller test-functions that only test part of the protocol. This will also make the protocol be more modular.